resources/config.yml

```diff
@@ -30,6 +30,18 @@ kicked-with-reason-message: "Kicked by admin.{line}Reason: {reason}"
 #-> uses: {line} {reason} {expires} {source}
 you-are-banned-logout: "§cYou are banned on this server!{line}§7Reason: §e{reason}{line}§7Expires at: §e{expires}"
 
+# Date Format syntax for Timer Durations
+# This uses class called DateInterval::diff()
+# Checkout this format for more information to how to make your own dateformat parameters.
+# https://www.php.net/manual/en/dateinterval.format.php#refsect1-dateinterval.format-parameters
+# The following characters are recognized in the format parameter string. 
+# Each format character must be prefixed by a percent sign (%).
+dateformat: "%y years, %m months, %d days, %h hours, %i minutes"
```

Comment:
Please use `{PREFIX}` instead of `%PREFIX`

Reply:
Please see this article about dateinterval formatting. https://www.php.net/manual/en/dateinterval.format.php#refsect1-dateinterval.format-parameters

In case of these, i need to change something like:
```diff
- dateformat: "%y years, %m months, %d days, %h hours, %i minutes"
+ dateformat: "{YEAR} year(s), {MONTHS} month(s), {DAY} day(s), {MINUTE} minute(s)"
```
These i need to replace all formatting codes from `DateTimeInterval::format()` to by using `preg_replace`